js/viewer.js

```diff
@@ -163,8 +163,13 @@ function redirectImages(node) {
 
 function applyAnchor() {
     var anchors = document.getElementsByName(setup.anchor);
-    if (anchors.length > 0)
+    if (anchors.length > 0) {
         anchors[0].scrollIntoView(true);
+        if (!local)
+          window.scrollBy(0, -46); // 46 is the height of the header of Cyberbotics web page
+        else
+          window.scrollBy(0, 180); // manual adjustment for the off-line version
```

Comment:
This hard-coded value (180) seems strange. It seems that the issue is not understood correctly.
Note also there are several types of anchors, at least the titles, the API and the figures.